release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -538,20 +540,43 @@ git push origin merged-master-release-[current release]
 
 Now create a pull request to merge the new branch you've made into the `release-[current-release]` branch on [k/website](https://github.com/kubernetes/website).
 
-### Freeze Kubernetes website
+#### Update the `config.toml`s for the past four releases
 
-24 hours before the release, freeze the repo: ⚠️  no PRs should be allowed to merge AT ALL until the release PR has successfully merged.
+Update `config.toml` files for the 4 previous releases. These need to be 4 separate PRs because each release has its own `release-` branch.
+Use [path release](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md) to determine the
+correct patch version of the past release when updating the `config.toml` file.
 
-- Request temporary write access to k/website. Any [SIG Docs co-chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) should be able to help with this. 
-- Submit an issue with `tide/merge-blocker` label. Depending upon your permissions, a [SIG Docs chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) can assist you with adding the label.
-- Submit a freeze announcement following our [protocols](#communicate-major-deadlines)
+See this for example (1.20 was the "future release"):
+* 1.16 https://github.com/kubernetes/website/pull/25392
+* 1.17 https://github.com/kubernetes/website/pull/25391
+* 1.18 https://github.com/kubernetes/website/pull/25390
+* 1.19 https://github.com/kubernetes/website/pull/25394
+
+⚠️  DO NOT MERGE **ANY** OF THE `config.toml` PULL REQUESTS FOR PREVIOUS UNTIL THE RELEASE IS CONFIRMED
```

Comment:
nit: move PREVIOUS?
DO NOT ... configuration PULL REQUESTS UNTIL THE RELEASE HAS OCCURRED [or completed]